packages/vsce/src/doc/resources/IJVMServer.ts

```diff
@@ -0,0 +1,6 @@
+import { IResource } from "./IResource";
+
+export interface IJVMServer extends IResource {
+    name: string;
+    enableStatus: string;
```

Comment:
This has to be lowercase to match what comes back from CMCI. Otherwise we get issues like this...
```suggestion
    enablestatus: string;
```

<img width="610" height="95" alt="Screenshot 2025-07-30 at 12 41 39 PM" src="https://github.com/user-attachments/assets/7c06161b-ac9e-4942-bb08-88bebc0f1117" />